Makefile

```diff
@@ -334,20 +334,20 @@ ci-job-clippy:
 
 define ci_setup_markdown_toc
 	$(call banner,CI-Setup: Install markdown-toc)
-	npm install -g markdown-toc
+	npm install markdown-toc
```

Comment:
This needs to be installed outside of the Tock repository, as having either the `node_modules` directory present breaks `markdown-toc`:

```
********************************************************************************
* CI-Job: Markdown Table of Contents Validation                                *
********************************************************************************

/home/runner/work/tock/tock/node_modules/markdown-toc/lib/insert.js:40
    throw new Error('markdown-toc only supports one Table of Contents per file.');
    ^

Error: markdown-toc only supports one Table of Contents per file.
    at Function.insert (/home/runner/work/tock/tock/node_modules/markdown-toc/lib/insert.js:40:11)
    at /home/runner/work/tock/tock/node_modules/markdown-toc/cli.js:59:27
```

Reply:
Installing one directory up seems to work fine on my local machine.

Reply:
Fixed!